BaseLib/CMakeLists.txt

```diff
@@ -15,6 +26,16 @@ if(MSVC)
 	target_link_libraries(BaseLib INTERFACE WinMM) # needed for timeGetTime
 endif()
 
+if(QT4_FOUND)
+	target_link_libraries(BaseLib PUBLIC Qt4::QtXml Qt4::QtXmlPatterns)
+	if(WIN32 AND CMAKE_CROSSCOMPILING)
+		find_package(OpenSSL)
```

Comment:
Put this in `scripts/cmake/Find.cmake`:

```cmake
find_package(OpenSSL)
```

And this into the `CMakeLists.txt`-file:

```cmake
if(WIN32 AND CMAKE_CROSSCOMPILING AND OPENSSL_FOUND)
    target_link_libraries(BaseLib PUBLIC Qt4::QtNetwork ${OPENSSL_LIBRARIES} ws2_32)
endif()
```